Firmware/variants/1_75mm_MK3S-EINSy10a-E3Dv6full.h

```diff
@@ -141,6 +141,11 @@
 #define DEFAULT_SAFETYTIMER_TIME_MINS 30
 #define FARM_DEFAULT_SAFETYTIMER_TIME_ms (45*60*1000ul)
 
+// Offline crash dumper
```

Comment:
Move closer to rest of DEBUG

Reply:
I kept it close to the "SAFETY" stuff in the hope that this is always going to be enabled...